pages/advanced/cli-reference/common-commands.mdx

```diff
@@ -61,7 +61,7 @@ send_ether <address> <value-wei> [--safe-nonce <int>]
 Sends an ERC-20 token from the Safe account to a different one.
 
 ```bash
-send_erc20 <address> <token_address> <value> [--safe-nonce <int>]
+send_erc20 <address> <token_address> <value-wei> [--safe-nonce <int>]
```

Comment:
To recheck now or in the future the use of `-` and `_`. It's clear for the commands, but not for the `<v_a-r_i-a_b-l_e-s>` :)
```suggestion
send_erc20 <address> <token-address> <value-wei> [--safe-nonce <int>]
```